vllm/worker/enc_dec_model_runner.py

```diff
@@ -280,19 +280,19 @@ def profile_run(self) -> None:
         for group_id in range(max_num_seqs):
             seq_len = (max_num_batched_tokens // max_num_seqs +
                        (group_id < max_num_batched_tokens % max_num_seqs))
+            seq_len = min(seq_len, self.model_config.max_model_len)
```

Comment:
Hmm, I think `seq_len` should never be calculated to be greater than `max_model_len` in the first place.